layer/USAGE_desktop.md

```diff
@@ -1,106 +1,142 @@
-# Using the Vulkan GFXReconstruct Layer on Desktop
+# Using the GFXReconstruct Layer on Desktop
```

Comment:
Would this be better as:

Using the GFXReconstruct Layer on the Desktop

(i.e. with "the" added)?